contracts/build.rs

```diff
@@ -76,6 +77,14 @@ fn main() {
 fn generate_contract(
     name: &str,
     deployment_overrides: HashMap<u32, (Address, Option<TransactionHash>)>,
+) {
+    generate_contract_with_config(name, deployment_overrides, |builder| builder)
+}
+
+fn generate_contract_with_config(
+    name: &str,
+    deployment_overrides: HashMap<u32, (Address, Option<TransactionHash>)>,
+    config: impl FnOnce(Builder) -> Builder,
```

Comment:
Nbd, but maybe deployment overrides and remaining config should be handled the way?

Reply:
Great idea! Turned out quite nice I think :smile: